aldor/aldor/lib/java/src/foamj/Foam.java

```diff
@@ -19,11 +37,26 @@ public static Word fgetss(Word w1, Word w2, Word w3, Word w4) {
     }
 
     public static Word fgetc(Word cw) {
-        throw new RuntimeException();
+        InputStream instream = Word.U.<InputStream>toJavaObj(cw);
+	try {
```

Comment:
This is not aligned with the above. Maybe we should have some auto-formatting to avoid these.